app/experimenter/experiments/tasks.py

```diff
@@ -210,6 +210,23 @@ def send_period_ending_emails(experiment):
                 "Sent ending email for Experiment: {}".format(experiment)
             )
 
+    # send enrollment ending emails if enrollment end
+    # date is 5 days out
+    if experiment.enrollment_end_date:
+        if (experiment.enrollment_end_date - date.today()) <= timedelta(
+            days=5
+        ):
+            if not ExperimentEmail.objects.filter(
+                experiment=experiment,
+                type=ExperimentConstants.EXPERIMENT_PAUSES,
+            ):
```

Comment:
This can help make the db query faster/simpler:

https://docs.djangoproject.com/en/2.2/ref/models/querysets/#exists